releasenotes/notes/pin-deps-dbe52c49e945daf5.yaml

```
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    All Ansible dependencies are now pinned to specific versions. The
+    ``stackhpc.vxlan`` role was pinned to 1.1.0, and ``ansible-role-docker``
+    was pinned to stackhpc/7.0.1.1.
```

Very nitpicky, but the tense here is inconsistent
```suggestion
    All Ansible dependencies are now pinned to specific versions. The
    ``stackhpc.vxlan`` role is pinned to 1.1.0, and ``ansible-role-docker``
    is pinned to stackhpc/7.0.1.1.
```